integrations/GA4/browser.js

```diff
@@ -33,26 +34,22 @@ export default class GA4 {
         window.dataLayer.push(arguments);
       };
     window.gtag("js", new Date());
-
+    const gtagParameterObject = {};
     // This condition is not working, even after disabling page view
     // page_view is even getting called on page load
     if (this.blockPageView) {
-      if (this.sendUserId) {
-        window.gtag("config", measurementId, {
-          user_id: userId,
-          send_page_view: false,
-        });
-      } else {
-        window.gtag("config", measurementId, {
-          send_page_view: false,
-        });
-      }
-    } else if (this.sendUserId) {
-      window.gtag("config", measurementId, {
-        user_id: userId,
-      });
-    } else {
+      gtagParameterObject.send_page_view = false;
+    }
+    if (this.sendUserId) {
+      gtagParameterObject.user_id = userId;
+    }
+    if (this.debugMode) {
+      gtagParameterObject.debug_mode = true;
+    }
+    if (Object.keys(gtagParameterObject).length === 0) {
       window.gtag("config", measurementId);
```

Comment:
Does it disallow sending empty objects in the 3rd parameter?

Reply:
debug_mode: !!this.debugMode

When debug_mode is `false` (i.e the toggle is off), we don't have to include debug_mode altogether, as then it will be treated as debug_mode : `true`

[Reference](https://support.google.com/analytics/answer/7201382?hl=en#zippy=%2Cglobal-site-tag-websites:~:text=To%20disable%20debug%20mode%2C%20exclude%20the%20%27debug_mode%27%20parameter%3B%20setting%20the%20parameter%20to%20false%20doesn%27t%20disable%20debug%20mode.)

Reply:
Got it. Weird.

Reply:
Please add this reference and info in the code.

Reply:
Sure. Added